esp/esp/program/modules/handlers/onsitecheckinmodule.py

```diff
@@ -76,7 +76,7 @@ def create_record(self, event):
                                                      program=self.program)
         return created
 
-    def delete_record(self, extension):
+    def delete_record(self, event):
```

Comment:
I think the (non-barcode, non-rapid) checkin module has been failing if you *remove* any checkin steps since the datatree merge.